livestream/views.py

```diff
@@ -0,0 +1,20 @@
+"""Livestream Views"""
+from rest_framework import status
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.response import Response
+
+from django.conf import settings
```

Comment:
This should be grouped with the `rest_framework` imports since they're both 3rd party libraries